deploy/nfs/kubernetes/csi-nfsplugin.yaml

```diff
@@ -2,154 +2,134 @@
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: csi-nfs-node
+  name: csi-nfsplugin
```

Comment:
Document the upgrade scenario as we will have new daemonset and users can delete old daemonsets

Reply:
https://github.com/ceph/ceph-csi/issues/3271